Tests/Shared.Specs/BasicEquivalencySpecs.cs

```diff
@@ -4193,14 +4200,17 @@ public class ClassWithAllAccessModifiersForMembers
         protected internal string ProtectedInternalProperty { get; set; }
         private string PrivateProperty { get; set; }
 
+        private protected string PrivateProtectedProperty { get; set; }
+
         public ClassWithAllAccessModifiersForMembers(string publicValue, string protectedValue, string internalValue,
-            string protectedInternalValue, string privateValue)
+            string protectedInternalValue, string privateValue, string privateProtected)
```

Comment:
🙃 Rename the parameter to `privateProtectedValue`

Reply:
Good catch, thanks.